lib/pagy/version.rb

```diff
@@ -0,0 +1,3 @@
+class Pagy
+  VERSION = '0.8.1'
+end
```

Comment:
I hate having a ruby file for the version

Reply:
needed for https://github.com/ddnexus/pagy/pull/29 and kinda a best-practice ...

alternatives:
 - have verison copied in gemspec and lib
 - do not use `gemspec` in Gemfile